### PR TITLE
Add max_priority to give priority ability to a queue

### PR DESCRIPTION
**Pull request by hyperized**
##### SUMMARY
Add max_priority to give priority ability to a queue.

https://www.rabbitmq.com/priority.html

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
modules/messaging/rabbitmq_queue.py

##### ANSIBLE VERSION

```
ansible 2.3.2.0
```

##### ADDITIONAL INFORMATION
n/a

### CODE REVIEW

File: lib/ansible/modules/messaging/rabbitmq_queue.py
Comment (abadger) [inline lib/ansible/modules/messaging/rabbitmq_queue.py]: Missing colon

File: lib/ansible/modules/messaging/rabbitmq_queue.py
Comment (abadger) [inline lib/ansible/modules/messaging/rabbitmq_queue.py]: You'll need a ```version_added: "2.4"``` and default: None here as well

**Comment by abadger** [comment]
Some comments on the documentation (which should solve the shippable errors).

The code looks good to me but I don't use rabbitmq.  Really need a +1 from one of the module owners (or failing that, a +1 from someone who tests out the new functionality thoroughly).  Once that happens, ping me on irc (abadger1999) and I'll merge.

**Comment by hyperized** [comment]
Hi @chrishoffman , @manuel-sousa  and @romanek-adam, could you have a look too :)?

File: lib/ansible/modules/messaging/rabbitmq_queue.py
Comment (pilou-) [inline lib/ansible/modules/messaging/rabbitmq_queue.py:107]: Parameters with default values could be omitted (default value for `required` is `false` and default value for `default` is `null`, `required` and `default` could be removed). As other parameters specify `required` and `default`, you might want to keep them in order to preserve consistency.

**Review by pilou- — APPROVED** [review]
Looks good to me

**Comment by pilou-** [comment]
@retr0h as the only other maintainer of namespace `messaging`, could you review this pull request ?

**Comment by hyperized** [comment]
shipit

**Comment by abadger** [comment]
Merged to devel for 2.5.x.  Thanks @pilou- for the review and @hyperized for the change!

**Pull request merged**
